src/sstruct_ls/node_relax.c

```diff
@@ -764,10 +764,8 @@ hypre_NodeRelax(  void                 *relax_vdata,
                    * Invert intra-nodal coupling
                    *----------------------------------------------*/
                   hypre_gselim(A_loc, x_loc, nvars, err);
-                  if (err)
-                  {
-                     hypre_error_w_msg(HYPRE_ERROR_GENERIC, "Error in gselim!\n");
-                  }
+                  (void) err;
```

Comment:
Hi @victorapm .  Sorry for breaking this.  Didn't realize it was device code.  And you are correct that we need to devise a way to capture errors on the device.

Can you add a comment after each of these `(voide) err;` lines that says something like, "need a way to check error codes on device"?  Otherwise, this looks fine to me and still compiles on the Mac.  Thanks!

Reply:
Sure, no worries! I can do that :) BTW, the `void (err)` line was to avoid the set-but-not-used warning :)

Reply:
Yep, I figured that.  And it works.

Reply:
That works but just when `HYPRE_DEBUG` is defined. We might need something in release mode as well. Let's discuss :)